grammar.ebnf

```diff
@@ -20,8 +20,19 @@ StructDecl = "struct", Identifier, "{", { StructField }, "}";
 StructField = Identifier, ":", Type, ";";
 
 
+Statement = "{", { Statement }, "}"
+          | "while", "(", Expression, ")", Statement
+          | Identifier, "=", Expression, ";"
+          | Identifier, "[", Expression, "]", "=", Expression, ";"
+          | "if", "(", Expression, ")", Statement
+          | Identifier, "(", FuncCallArgList, ")", ";" 
+          | "return", Expression, ";";
+
+FuncCallArgList = [Identifier, { ",", Identifier }];
```

Comment:
This is unfortunately not correct. Say you write this call: `a(1, 2)`.

Reply:
Oops, silly mistake. Will refrain from posting so eagerly :p